### ISSUE TITLE
Recommended method of installation

### ISSUE THREAD

**Issue by lhk** [jupyter/notebook#462]
The jupyter website recommends to install anaconda in order to get jupyter up and running.

I did a clean install of anaconda (for python 3.4) with the latest sh script of their website, added the bin folder to my Path and executed:

    conda update conda
    conda update anaconda
    conda update python
    conda install jupyter
    conda update jupyter

now jupyter is available in my shell, I can start both notebooks in the browser and as qtconsoles.

But there is a problem: It is impossible to import pyplot from matplotlib. Here is the pastebin http://pastebin.com/RJ0Mj2F0

I tried to install pygobject with pip, but apparently anaconda has its own copy of the python interpreter. Pygobject or pygtk are not in the anaconda channels.

This command:
    anaconda search -t conda pygtk

delivers:
    
    Using binstar api site https://api.anaconda.org
    Run 'anaconda show <USER/PACKAGE>' to get more details:
    Packages:
         Name                      |  Version | Package Types   | Platforms      
         ------------------------- |   ------ | --------------- | ---------------
         auto/pygtk-textbuffer-with-undo | 0.1.post2.dev74151128 | conda           | linux-64       
                                              : http://github.com/cfobel/pygtk_textbuffer_with_undo.git
         auto/pygtkhelpers         |    0.4.3 | conda           | linux-64       
                                              : http://packages.python.org/pygtkhelpers
         mike/pygtk-aio            | 2.24.32.win32.py2.7 | conda           | win-32         
                                              : PyGtk All In One
         ska/pygtk                 |   2.16.0 | conda           | linux-64       
                                              : Pipeline running tools
         vgauthier/pygtk           |   2.24.0 | conda           | osx-64         
                                              : PYGTK
    Found 5 packages

It seems like only ska/pygtk is supported on linux, but that is an outdated version of pygtk. The current version is 2.24.

What is the correct way to install jupyter ?
It would be great if you could add more information to your website. It seems like installation is straightforward, but without pyplot this is hardly usable.

**Comment by lhk** [comment]
I tried to install the outdated package, but there is a conflict with python 3.4
    conda install -c https://conda.binstar.org/ska pygtk 
    Fetching package metadata: ......
    Solving package specifications: ..
    Error: Unsatisfiable package specifications.
    Generating hint: 
    [      COMPLETE          ]|############################################################################## ############################################################################################################################| 100%


    Hint: the following packages conflict with each other:
      - pygtk
      - python 3.4*

    Use 'conda info pygtk' etc. to see the dependencies for each package.

    Note that the following features are enabled:
      - gtk

**Comment by willingc** [comment]
@lhk From the first pastebin, it looks as if the last error message indicates that pygobject should be installed. ``ImportError: Gtk3 backend requires pygobject to be installed.``

Searching at anaconda.org, it looks as if there is a third party option conda option for installing pygobject at https://anaconda.org/fallen/pygobject. To install, ``conda install -c https://conda.anaconda.org/fallen pygobject``.

You may wish to ``conda remove`` the pygtk package that you installed to avoid a conflict.

I hope this helps. I'm on OS X so I can't test the linux easily for you. Good luck.

**Comment by Carreau** [comment]
Well yes that seem to be (one of ) the correct way to install jupyter, which in itself works.
The issue is with the specific Gtk3 backend of matplotlib (which is not a jupyter issue). 

Can you re-try by starting your notebook with `%matplotlib inline` ? It might prevent matplotlib to try to import the gtk3 backend. 
let see if that works. If it does we can also redirect you to the matplotlib and continuum developpers to see if they can help you to also have the gtk3 backend to works.

**Comment by lhk** [comment]
Thank you very much
First I tried inlining the output. That solves the problem
Then I tried the pygobject package. Now I can import the library, but it doesn't seem to work.

I tried this sample http://matplotlib.org/examples/shapes_and_collections/scatter_demo.html

With the inlining option, the scatterplot is drawn correctly.
With the pygobject package, a window opens to display the graph. But it stays empty.
